runtime-std/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "polkadot-runtime-std"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+build = "build.rs"
+
+[build-dependencies]
+rustc_version = "0.2"
+
+[dependencies]
+pwasm-alloc = { path = "../wasm-runtime/pwasm-alloc", version = "0.1" }
+pwasm-libc = { path = "../wasm-runtime/pwasm-libc", version = "0.1" }
+environmental = { path = "../environmental", version = "0.1", optional = true }
+polkadot-state-machine = { path = "../state-machine", version = "0.1", optional = true }
+polkadot-primitives = { path = "../primitives", version = "0.1", default_features = false }
+polkadot-runtime-codec = { path = "../runtime-codec", version = "0.1", default_features = false }
+triehash = { version = "0.1", optional = true }
```

Comment:
quite a few more (wasm) dependencies here than before. triehash and ed25519 may not be super efficient in wasm, hence the native externs

Reply:
the optional deps are only linked when compiling with std

Reply:
cool.